pom.xml

```diff
@@ -35,6 +35,7 @@
 
         <impsort-maven-plugin.goal>sort</impsort-maven-plugin.goal>
 
+        <disruptor.version>3.3.4</disruptor.version>
```

Comment:
Is there a particular reason to use the older 3.X series? Log4j suggests it [supports 4.X](https://logging.apache.org/log4j/2.x/manual/async.html#installation)

Reply:
No, we can use 4.X if it's supported, the docs I was looking at just didn't mention it.